src/olympia/activity/models.py

```diff
@@ -150,6 +151,18 @@ class Meta:
         ordering = ('-created',)
 
 
+class ReviewActionReasonLog(ModelBase):
+    """
+    This table allows ReviewActionReasons to be assigned to ActivityLog entries.
+    """
+
+    activity_log = models.ForeignKey('ActivityLog', on_delete=models.CASCADE)
+    reason = models.ForeignKey(ReviewActionReason, on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ('-created',)
+
```

Comment:
Activity log models are one of the few places we still specify `db_table`, to have them consistently all named `log_activity_xxx`